### PR TITLE
Refactor bootstrapping in project templates

### PR DESCRIPTION
**Pull request by dschissler**

### CODE REVIEW

**Comment by dschissler** [comment]
Damn, I just forked the repository and it didn't tell me that I already had a repository.  It just took me to my existing repository....  Annoying.  So what to do @sergeyklay?

File: templates/project/micro/config.php [hunk 1]
```diff
@@ -1,7 +1,5 @@
 <?php
 
-defined('APP_PATH') || define('APP_PATH', realpath('.'));
-
```
Comment (sergeyklay) [inline templates/project/micro/config.php]: @dschissler Why?

Comment (dschissler) [inline templates/project/micro/config.php]: Why wouldn't it be set already?  I don't feel strongly about that so it can just be added back.

Comment (sergeyklay) [inline templates/project/micro/config.php]: For example we can use Devtools with Micro app. Devtools starts from `public` dir

Comment (dschissler) [inline templates/project/micro/config.php]: But I think that the index.php there which is sort of a all-in-one bootstrap already defines it.

Comment (sergeyklay) [inline templates/project/micro/config.php]: By default Webtools does not use index.php

Comment (dschissler) [inline templates/project/micro/config.php]: I didn't test it with Webtools.

**Comment by sergeyklay** [comment]
Thanks

**Pull request merged**